src/app/ApproveActivities/components/ApproveActivitiesComponent.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import dateFns from 'date-fns';
+import PropTypes from 'prop-types';
+
+import ActionsComponent from '../../VerifyActivities/components/ActionsComponent';
+import { TableComponent, TruncateDescriptionContainer } from '../../common/components';
+
+const ApproveActivitiesComponent = ({ activities }) => {
+  let tableBodyHtml;
+  const columnNames = ['Activity', 'Date', 'Points', 'Description', 'Actions'];
+  if (!activities.length) {
+    tableBodyHtml = (
+      <tr className='myactivities__table__row'>
+        <td colSpan={6} className='myactivities__table__data'>
+          There is no activities to approve
+        </td>
+      </tr>
+    );
+  } else {
+    tableBodyHtml = activities.map((activity) => {
+      const {
+        id, activityDate, points, description, category,
+      } = activity;
+      return (
+        <tr key={id} className='myactivities__table__row'>
+          <td>{category}</td>
+          <td>{dateFns.format(new Date(activityDate), 'MMM DD YYYY')}</td>
```

Comment:
The date format should be 'MMM dd yyyy', the current format might Invalid date format errors.